.github/RELEASE.md

```diff
@@ -0,0 +1,265 @@
+        -  `name`
+        -  `email`
+        -  `github`
+        -  `aliases`
+        -  `alternate_emails`
+
+      Other keys (e.g., `num_commits` and `first_commit`) do not need to be included in this separate commit as they will be overwritten by rever.
+
+    - Here's a sample run where we undo the commit made by rever in order to commit the changes to `.authors.yml` separately:
+
+        ```bash
+        (rever) $ rever --activities authors 22.9.0 --force
+
+        # changes were made to .authors.yml as per the prior bullet
+        (rever) $ git diff --name-only HEAD HEAD~1
+        .authors.yml
+        .mailmap
+        AUTHORS.md
+
+        # undo commit
+        (rever) $ git reset HEAD~1 --soft
+
+        # undo changes made to everything except .authors.yml
+        (rever) $ git restore .mailmap AUTHORS.md
+        ```
+
+    - Commit these changes to `.authors.yml`:
+
+        ```bash
+        (rever) $ git add .
+        (rever) $ git commit -m "Updated .authors.yml"
+        ```
+
+    - Rerun `rever --activities authors` and finally check that your `.mailmap` is correct by running:
+
+        ```bash
+        git shortlog -se
+        ```
+
+      Compare this list with `AUTHORS.md`. If they have any discrepancies, additional modifications to `.authors.yml` is needed, so repeat the above steps as needed.
+
+    - Once you are pleased with how the author's file looks, we want to undo the rever commit and commit the `.mailmap` changes separately:
+
+        ```bash
+        # undo commit (but preserve changes)
+        (rever) $ git reset --soft HEAD~1
+
+        # undo changes made to everything except .mailmap
+        (rever) $ git restore .authors.yml AUTHORS.md
+        ```
+
+    - Commit these changes to `.mailmap`:
+
+        ```bash
+        (rever) $ git add .
+        (rever) $ git commit -m "Updated .mailmap"
+        ```
+
+    - Continue repeating the above processes until the `.authors.yml` and `.mailmap` are corrected to your liking. After completing this, you will have at most two commits on your release branch:
+
+        ```bash
+        (rever) $ git cherry -v main
+        + 86957814cf235879498ed7806029b8ff5f400034 Updated .authors.yml
+        + 3ec7491f2f58494a62f1491987d66f499f8113ad Updated .mailmap
+        ```
+
+
+4. Review news snippets (ensure they are all using the correct Markdown format, **not** reStructuredTest) and add additional snippets for undocumented PRs/changes as necessary.
+
+    > **Note:**
+    > We've found it useful to name news snippets with the following format: `<PR #>-<DESCRIPTIVE SLUG>`.
+    >
+    > We've also found that we like to include the PR #s inline with the text itself, e.g.:
+    >
+    > ```markdown
+    > ### Enhancements
+    >
+    > * Add `win-arm64` as a known platform (subdir). (#11778)
+    > ```
+
+    - I find it helpful to use [GitHub's compare view][compare] to review what changes are to be included in this release.
```

Comment:
Just a nitpick, I found the first-person voice here to not quite fit in to the style of the rest of the document (this is an optional "take it or leave it" suggestion):
```suggestion
    - You can utilize [GitHub's compare view][compare] to review what changes are to be included in this release.
```